src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/TakeCard.kt

```diff
@@ -21,53 +18,73 @@ import tornadofx.*
 
 class TakeCard(val take: Take, player: IAudioPlayer) : AnchorPane() {
     val playedProperty = SimpleBooleanProperty(take.played)
-    val badge = stackpane {
-        addClass(TakeCardStyles.badge)
-        val icon = TakeCardStyles.badgeIcon()
-        icon.addClass(TakeCardStyles.iconStyle)
-        add(icon)
-        isVisible = !take.played
-    }
 
     var deleteButton: Button by singleAssign()
     var takeNumberLabel: Label by singleAssign()
     var timestampLabel: Label by singleAssign()
+    var playButton: Button by singleAssign()
+    var editButton: Button by singleAssign()
+
+
     var simpleAudioPlayer: SimpleAudioPlayer by singleAssign()
```

Comment:
audioPlayer or even just player would be a better name for the member variable honestly

Reply:
do you meant the entire  `SimpleAudioPlayer.kt` file or just the declaration here?

Reply:
as per @natibekele, name was to distinguish from some future waveform implementation, closing issue as it's fine